lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -635,6 +635,56 @@ class ConvertAtenSqueezeAllDimsOp : public ConvertAtenSqueezeOp<AtenOpT> {
   }
 };
 
+// FIXME(AG): This will eventually go into a Tosa*Utils file
+// Convert an fp32 scalar into tosa fp32 tensor.
+static LogicalResult tosaF32TensorFromTorchFloat(ConversionPatternRewriter &rewriter,
+                                        Operation *op, Value torchScalarValue,
+                                        Value &tosaTensor) {
+  double scalarValue;
+  // Check if this is an unrealized conversion cast
+  auto uConvCast = dyn_cast<mlir::UnrealizedConversionCastOp>(torchScalarValue.getDefiningOp());
+  auto uConvCastInOps = uConvCast.inputs();
+  // Ensure that there is exactly one input
+  if (uConvCastInOps.front() != uConvCastInOps.back()) op->emitError("This cast must have exactly one operand");
```

Comment:
the if block need to be in a new line. Please use `clang-format` to format the code.